apps/backend/src/app/modules/maps/maps.controller.ts

```diff
@@ -189,6 +190,18 @@ export class MapsController {
     return this.mapsService.getAll(query, userID);
   }
 
+  @Get('/preSignedUrl')
```

Comment:
Weird name, let's do `getMapUploadUrl`. I know that's a bit pedantic but should be an easy find-and-replace.